e2e_samples/parking_sensors/README.md

```diff
@@ -230,70 +255,72 @@ More resources:
 - [jq](https://stedolan.github.io/jq/)
 - [makepasswd](https://manpages.debian.org/stretch/makepasswd/makepasswd.1.en.html)
 
-#### Software pre-requisites if you use dev container
+### Configuration and Deployment
 
-- [Docker](https://www.docker.com/)
-- [VSCode](https://code.visualstudio.com/)
-- [Visual Studio Code Remote Development Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
-
-  It is strongly recommended to use dev container for the deployment to avoid environment related issues.
-
-### Setup and Deployment
+Proceed in configure the environment variables as outlined below, fork the GitHub repository, and log into your Azure account before proceeding with the deployment.
```

Comment:
do we need this sentence?

Reply:
So I am just adding sentences to introduce topics. I believe it is more organize and clear, instead of having a topic and nothing said.

Reply:
it seems a bit of a repeat. if you want to keep this, can you please rephrase as it's not grammatically correct right now (maybe instead of "Proceed in configure" you meant "proceed with configuring"?)

Reply:
yeahp.... I can do that... no worries...I will rephrase.